jobs/grafana/templates/bin/influxdb-datasource

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -eu
+
+GRAFANA_DATASOURCES_API="<%= p('grafana.server.protocol', 'http') %>://<%= spec.ip %>:<%= p('grafana.server.http_port') %>/api/datasources"
+GRAFANA_CREDENTIALS="<%= p('grafana.security.admin_user') %>:<%= p('grafana.security.admin_password') %>"
+DATASOURCE_ID=""
+
+<% if p('grafana.influxdb.url') %>
```

Comment:
when properties are optional, instead of `<% if p(...) %>` it's better to use `<% if_p(...) do |var|`.